src/style.css

```diff
@@ -1 +1,136 @@
-/* Styles go here */
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(fonts/Roboto-Medium.woff);
+}
+
+.html {
+  box-sizing: border-box;
+
+  font-family: Roboto, sans-serif;
+}
+
+/*
+  Чому я використовую бокс-сайзінг таким чином,
+  замість звичайного * {...}, тому що деякі бібліотеки,
+  по типу Swiper, використовують box-sizing: content-box
+  (наприклад класс swiper-wrapper) і, щоб бібліотека відображалась так,
+  як це задумував розробник, їй потрібно залишити box-sizing,
+  який вказав розробник. Тому я використовую наслідування.
+*/
+
+* {
+  box-sizing: inherit;
+}
+
+.body {
+  margin: 0;
+}
+
+.link-reset {
+  color: inherit;
+  text-decoration: none;
+}
+
+.list-reset {
+  margin: 0;
+  padding: 0;
+
+  /*
+    If we remove the visual list markers
+    from an ordered or unordered list with list-style-type: none;,
+    we’re likely to run into some issues with VoiceOver,
+    Apple’s screen reader software, so there is invisible marker below
+  */
```

Comment:
Great, I don't think that it's critical, but still it's great that you've mentioned that